packages/slice-machine/server/src/api/common/hooks/updateLibraries.ts

```diff
@@ -59,7 +59,7 @@ const createIndexFileForFrameWork = (
   return createIndexFile(lib);
 };
 
-export default async function onSaveSlice(
+export default async function reGenerateLibrariesState(
```

Comment:
libraryStates.json is a very specific file in the file system (.slicemachine), i think it should be `generateLibrariesIndex)`